iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -62,8 +62,25 @@ PortManager::PortManager(RouDiMemoryInterface* roudiMemoryInterface) noexcept
     }
     auto introspectionMemoryManager = maybeIntrospectionMemoryManager.value();
 
+    popo::PublisherOptions registryPortOptions;
+    registryPortOptions.historyCapacity = 1U;
+    registryPortOptions.nodeName = iox::NodeName_t("Service Registry");
+    registryPortOptions.offerOnCreate = true;
+
+    m_serviceRegistryPublisherPortData =
+        acquirePublisherPortData(serviceRegistryService,
+                                 registryPortOptions,
+                                 IPC_CHANNEL_ROUDI_NAME,
+                                 introspectionMemoryManager,
+                                 PortConfigInfo())
+            .or_else([](auto&) {
+                LogError() << "Could not create PublisherPort for service registry!";
+                errorHandler(Error::kPORT_MANAGER__NO_PUBLISHER_PORT_FOR_SERVICE_REGISTRY);
```

Comment:
Please add the error level `Fatal` here and add a `LogFatal`.

When roudi is unable to create a publisher during startup then no application is able to do it during runtime and the system is broken either way.